File: C/52_Programming_Problem/big_to_small_number.c
/*
Problem is~
    
    Firstly, input who much number run this code. And then print number 1000 to 1 and 5 number in 1 line with single tap gap between two number
*/

#include<stdio.h>

int main(){

       for(int i=1000; i>0; i--){
              if(i%5 == 0){
                     printf("\n");
              }
              printf("%d\t", i);
       }

       return 0;
}

File: Python/52_Programming_Problem/big_to_small_number.py
"""
Problem is~
    
    Firstly, input who much number run this code. And then print number 1000 to 1 and 5 number in 1 line with single tap gap between two number
"""

for i in range(1000, 0, -1):
    if i % 5 == 0:
        print()
    print(i, end="\t")
